strategy/rfp-to-videographers.md

```diff
@@ -1,154 +1,114 @@
+We believe that to achieve maximum distribution and impact, these videos should be produced by a professional studio. We would like to invite your company to submit a detailed proposal for the production of one of these videos, _A Coffee Place_ (working title).
 
-HTC members will assess your proposal based on your preliminary ideas, relevant reference projects and budget.
+HTC members will assess your proposal on the basis of your preliminary ideas, experience with similar projects, schedule, and budget. As you will understand, the HTC as a community of volunteers will pay special attention to the budget you propose. We hope that the final deliverable can be made without sacrificing the high quality needed to reach a large audience.
 
-As you will understand, the HTC as a community of volunteers will pay special attention to any proposal you can make with regard to budget, without sacrificing the desired quality of the final deliverable.
+If your company sympathizes with our cause, or believes, as we do, that this project has a high potential to be widely distributed and reach millions, we also invite you to consider working with us on a partnership basis, where you will essentially produce the video for free, in exchange for special credits and other advantages to be discussed. 
 
-If your company sympathizes with our cause, or believes, as we do, that this project has a very high potential to be widely distributed and reach millions, we also invite you to consider working with us on a partnership basis, where you will essentially produce the video for free, in exchange for special credits and other advantages to be discussed. 
+You will find attached the following information to help you prepare your proposal:
 
-You will find in appendix the following information to help you prepare your proposal:
+1. Campaign theme, description, and objectives
+2. Video concept, script, and objectives
+3. Scope of work
+4. Key issues
+5. Elements of your proposal
 
-- Campaign theme and objectives
-- Campaign description and objectives
-- Video concept, script and objectives
-- Scope of work
-- Key issues
-- Structure of your proposal
-
-Should you have any query on this project, please contact anyone of the following HTC representatives:
+Should you have any questions about this project, please contact one of the following HTC representatives:
 
 - Contact 1
 - Contact 2
 - Contact 3
 
-We hope you will find this project as exciting as we do and look forward to working with you.
+We hope you will find this project as exciting as we do. It would be best if we could hear from you by November 30, 2019, but if more time is needed, please let us know.
 
 Best regards,
 [Name]
 Humane Technology Community
 
-### Appendix
-
-#### "Mankind 2.0" Campaign Theme
```

Comment:
Good that you have left out the Theme document. This will indeed be too much, and RFP receiver does not know the structure of `Theme` > `Campaign`. I see you created a smaller summary.

Only thing to be aware of - when creating an easy-to-use RFP template from this - is that RFP creation should not have too many changes to the original proposal. Ideally this inclusion is a simple copy / paste action.